src/commonMain/kotlin/com/airthings/lib/logging/LogDate.kt

```diff
@@ -52,9 +51,13 @@ data class LogDate(
     override fun equals(other: Any?): Boolean =
         other is LogDate && other.year == year && other.month == month && other.day == day
 
-    override fun toString(): String = toString(separator)
+    override fun toString(): String = toString(SEPARATOR)
 
     override fun hashCode(): Int = toString(null).toIntOrNull() ?: toString().hashCode()
+
+    companion object {
+        const val SEPARATOR = '-'
```

Comment:
As we're now a OSS project, we'll need to add proper documentation and run ktlint and detekt to ensure there are no linting mistakes.

```suggestion
        /**
         * The character used to separate date parts, f.ex: `2023-08-23`.
         */
        const val SEPARATOR = '-'
```

Reply:
✅